zrml/prediction-markets/src/lib.rs

```diff
@@ -221,11 +179,8 @@ mod pallet {
         /// market to resolved.
         ////
         #[pallet::weight((T::WeightInfo::admin_move_market_to_resolved_overhead()
-            .saturating_add(T::WeightInfo::internal_resolve_categorical_reported(
-                4_200,
-                4_200,
-                T::MaxCategories::get().into()
-            ).saturating_sub(T::WeightInfo::internal_resolve_scalar_reported())
+            .saturating_add(T::WeightInfo::internal_resolve_categorical_reported()
+            .saturating_sub(T::WeightInfo::internal_resolve_scalar_reported())
```

Comment:
Do you know why this is `saturating_sub` here?

Reply:
As per comment on admin_move_market_to_resolved_overhead() benchmark it uses scalar market hence we does not substract internal_resolve_scalar_reported() then weight of on_resolution() for scalar market will be counted twice, first in admin_move_market_to_resolved_overhead() and then again in internal_resolve_categorical_reported() :thinking:

Reply:
I see. Okay!